awesome.js

```diff
@@ -0,0 +1,49 @@
+const decodedbooks = JSON.parse(localStorage.getItem('books'));
+const add = document.querySelector('.add');
+function showbook(books) {
+  if (books.length === 0) {
+    let inner = '';
+    inner += `<p>You have no record in the store
+    </p>`;
+    document.querySelector('.input-display').innerHTML = inner;
+  }
+  let html = '';
+  books.forEach((book, index) => {
+    html += `<p class="show-title">${book.title}</p>
+      <p class="show-author">${book.author}</p>
+      <span class="btn-span"><button class="rv-button" type="button" onclick="removeBook(${index})">Remove</button></span><br>
+      <hr>`;
+    document.querySelector('.input-display').innerHTML = html;
+  });
+}
+function removeBook(index) {
```

Comment:
Upon adding multiple books and removing one followed by adding a book, it resets previously added books and only the last added book persists. Kindly fix it. If you are not able to understand my statement, feel free to message me on slack.